tests/denops/testdata/dummy_interrupt_plugin.ts

```diff
@@ -1,9 +1,11 @@
 import type { Entrypoint } from "jsr:@denops/core@^7.0.0";
+import { delay } from "jsr:@std/async@^1.0.1/delay";
 
 export const main: Entrypoint = (denops) => {
   function reset(): void {
     const signal = denops.interrupted;
     signal?.addEventListener("abort", async () => {
+      await delay(100);
```

Comment:
:memo:

This delay is required to mimic slow abort process (so that "returns immediately" tests passed as expected)